internal/runbits/buildplanner/buildplanner.go

```diff
@@ -0,0 +1,173 @@
+package buildplanner
```

Comment:
This was extracted from the artifacts runner. It was fetching a commit and returning the commit's build plan. This runbit has two functions: GetCommit and GetBuildPlan. The logic is unchanged, but broken up into two functions.